docs/userguide/active_learning.md

```diff
@@ -0,0 +1,68 @@
+# Active Learning
+When labeling unmeasured data points, e.g. for a data acquisition campaign to gather 
+data for a machine learning model, it can be beneficial to follow a guided approach 
+rather than randomly measuring data. If this is done via iteratively measuring points 
+according to a criterion proportional to the current model's uncertainty, the 
+method is called `active learning`.
```

Comment:
Backticks are typically reserved for code, I'd prefer making this bold.

Reply:
done